test/rubygems/test_gem_specification.rb

```diff
@@ -2932,7 +2932,7 @@ def test_validate_license_values_plus
     util_setup_validate
 
     use_ui @ui do
-      @a1.licenses = ['GPL-2.0+']
+      @a1.licenses = ['MIT+']
```

Comment:
`MIT+` doen't make sense, because the MIT license is unversioned and there's no clear steward who would cut later releases.   `CDDL-1.0+` might be a better exampl (more background [here][1] and [here][2]).

[1]: https://wiki.spdx.org/view/Legal_Team/or-later-vs-unclear-disambiguation#Background_.28additional.29
[2]: https://wiki.spdx.org/view/Legal_Team/later-version-clauses

Reply:
Fixed in f81e6b2.